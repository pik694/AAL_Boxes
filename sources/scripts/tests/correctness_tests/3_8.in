3
41 99 8
51 72 11
16 90 32

