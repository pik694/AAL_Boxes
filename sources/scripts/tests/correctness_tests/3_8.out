0 1 2 
118944->118944
